.vscode/launch.json

```diff
@@ -4,8 +4,8 @@
         {
             "type": "chrome",
             "request": "launch",
-            "name": "Roxie DDL",
-            "url": "file:///${workspaceRoot}/demos/dermatology.html?src/marshaller/HTML.roxie",
+            "name": "Dermatology",
+            "url": "file:///${workspaceRoot}/demos/dermatology.html",
```

Comment:
This should be `"url": "file:///${workspaceRoot}/demos/dermatology/index.html",`

Reply:
I think your reviewing the wrong commit - as this is not in the final PR...